composer.json

```diff
@@ -2,6 +2,7 @@
   "name": "mysiar/omnipay-przelewy24v1",
   "type": "library",
   "description": "Przelewy24 API v1 driver for the Omnipay payment processing library",
+  "version": "1.0.16",
```

Comment:
nie wiem czy w tym miejscu ustawianie wersji jest odpowiednie, lepiej to przenieść do README
https://getcomposer.org/doc/articles/versions.md#composer-versions-vs-vcs-versions

Reply:
dołożyć do readme